### DISCUSSION TITLE
Update-2023-07-21 Curve colors

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#228] [Announcements]
Implemented curve colors and more Spirograph demos.

![Screen Shot 2023-07-21 at 12 38 24 PM](https://github.com/kaveh808/kons-9/assets/103860793/02e4f98e-c7ef-45d3-a722-8d12e9a4045d)

![Screen Shot 2023-07-21 at 1 05 34 PM](https://github.com/kaveh808/kons-9/assets/103860793/992135b9-a3b3-4b35-bf90-d1aa9c826c97)

**Comment by jolby** [comment]
This looks beautiful!

**Reply by aykaramba** [reply]
It really does.

**Comment by SlawickCoffe** [comment]
First one is an awesome!
